src/griffe/docstrings/google.py

```diff
@@ -32,13 +32,14 @@
     "params": DocstringSectionKind.parameters,
     "parameters": DocstringSectionKind.parameters,
     "keyword args": DocstringSectionKind.other_parameters,
-    "keyword parameters": DocstringSectionKind.other_parameters,
+    "keyword arguments": DocstringSectionKind.other_parameters,
     "other args": DocstringSectionKind.other_parameters,
     "other parameters": DocstringSectionKind.other_parameters,
     "raises": DocstringSectionKind.raises,
     "exceptions": DocstringSectionKind.raises,
     "returns": DocstringSectionKind.returns,
     "yields": DocstringSectionKind.yields,
+    "example": DocstringSectionKind.examples,
```

Comment:
I indeed consciously omitted `Example` because it is a common admonition type. I felt like only having `Examples` for special examples section was enough. So yeah, I'd like to keep it that way and you can remove this line :slightly_smiling_face: